### ISSUE TITLE
highlight new messages in post

### ISSUE THREAD

**Issue by jdorweiler** [duckduckgo/community-platform#458]
When you check your notifications it brings you to the forum post.  It's difficult to find the new post in a longer nested thread.

From a forum comment by @brendanheywood  "if it could highlight which comments where new since you last visited a page that would make a massive difference. It's very easy to lose track and context in these deeply nested threads"

**Comment by brendanheywood** [comment]
I think the simplest and most important thing is just deep linking into the comment.

* [ ] Specifically in the email I get, this link:

![image](https://cloud.githubusercontent.com/assets/187449/5770237/8017350c-9d7b-11e4-8e6c-7511ea269ec5.png)

That link goes to: https://duck.co/forum/comment/51533 which is a single comment completely out of context. Why does this page even exists? It's pointless. I've already seen that exact comment in the email and the whole point of the link is to see more context.

Really the link should go to:

https://duck.co/ideas/idea/4759/hiking-and-climbing-spots-near-a-location#comment_51533

There is already an id on the comment so the hash link is ready to but isn't being used!

* [ ] Similarly on the notifications page I see this:

![image](https://cloud.githubusercontent.com/assets/187449/5770260/dfc55e70-9d7b-11e4-8970-12d4b56f5191.png)

All three of those links go to the forum page without the hash link when you could trivially add the hash anchor and deep link directly to the comment. It's also unclear from that page which comments are older and newer and which should be read first.

**Comment by jdorweiler** [comment]
I think this would be a good to have.  It seems easy to do but I'm not too familiar with this part of the community platform.
@jbarrett what do you think?

**Comment by jbarrett** [comment]
Yeah, linking to the individual comment is not that useful. I suspect it's using the same template as comments in the forums etc, which has a link to the individual comment.

They should be switched out for the links to the thread + anchor, yes.

The xslate templates are kind of tough to navigate, but if I don't get round to making this change I can point you in the right direction - all the changes should be in there.
